rocky/tools/view_helpers.py

```diff
@@ -179,3 +181,10 @@ def schedule_task(request: HttpRequest, organization_code: str, task: QueuePrior
                 "It may take some time, a refresh of the page may be needed to show the results."
             ),
         )
+
+
+def get_scheduler_task(request: HttpRequest, organization_code: str, task_id: str) -> Optional[Task]:
```

Comment:
Im not sure this method is needed as a helper. We could just as easily catch the error in the view itself, where the request is also available, and where this message adding is more 'local' to the view

Reply:
Ahh I see that the error message is passed, the only thing it solves here is repeating code of try and catch in alle views using it.

Reply:
Yes, but this helper needs a reference to the requests, and then adds things to it (messages), while also silently catching errors that should be obvious to the developer fetching a task.

Reply:
I have remove it and do the exceptions where needed